### DISCUSSION TITLE
英特爾：想實現沉浸式元宇宙？運算效率還得提升 1 千倍

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#231] [General]
https://technews.tw/2021/12/22/immersive-metaverse-computing-efficiency/
晶片大廠英特爾（Intel）高層 Raja Koduri 上週發新聞稿表示，如果想創造真正持久且沉浸式元宇宙體驗，就算拿最先進技術相比，演算效率也需提高 1,000 倍。但他樂觀表示技術與軟體呈指數型成長，很快就能看到夢想實現。

**Comment by RogelioKG** [comment]
很現實的觀點，所以前陣子真的是吹得太過頭了。
網路與終端的協作都要配合得很好才能達成沉浸式的體驗，
現在頭戴式裝置的有線、FOV太小、PPD不足都會造成體驗的破壞，
還有不斷的網路延遲所造成的暈眩等等。
沉浸式體驗由最短的短板決定，
只要有任何一個問題無法解決，
就無法達到我們所追求的體驗。
我們還需要更多時間讓這項技術成熟。

**Comment by Chenszuhong** [comment]
很期待可以看到這項技術出來 雖然要克服很多困難 但是這會是進步到下一個時代的重要關鍵
